### PR TITLE
Pipeline tasks can return multiple outputs to unpack

### PR DESCRIPTION
**Pull request by jrs65**
- test(pipeline): ensure multiple output pipeline tasks are supported
- fix(pipeline): pipeline tasks can return multiple outputs to unpack

### CODE REVIEW

File: caput/tests/conftest.py [hunk 1]
```diff
@@ -124,6 +124,25 @@ def write_output(self, filename, output, file_format=None, **kwargs):
...
+multi_eggs_pipeline_conf = """
+---
+pipeline:
+  tasks:
+    - type: caput.tests.conftest.GetEggs
+      params: eggs_params
+      out: [color, egg]
+    - type: caput.tests.conftest.CookEggs
+      params: cook_params
+      in: egg
```
Comment (rikvl) [inline caput/tests/conftest.py:136]:
```suggestion
      in: color
```
Not 100% sure if I understood the intention of this test. Were the same eggs supposed to be fried twice? If the idea was to test a task with multiple outputs, where the outputs get used by subsequent tasks, then I think `color` should be an input somewhere.

File: caput/tests/test_pipeline.py [hunk 2]
```diff
@@ -8,3 +8,10 @@ def test_pipeline():
     result = conftest.run_pipeline()
     print(result.output)
     assert result.exit_code == 0
+
+
+def test_pipeline_multiple_outputs():
+    """Test running a very simple pipeline."""
```
Comment (rikvl) [inline caput/tests/test_pipeline.py:14]:
```suggestion
    """Test running a very simple pipeline with a multiple-outputs task."""
```

**Review by rikvl — CHANGES_REQUESTED** [review]
Hope I understood what's going on here. If so, I think this only requires one or two small changes.

**Comment by ljgray** [comment]
The key functionality here was added in a different PR, so this is now just adding tests

**Review by ketiltrout — APPROVED** [review]
This seems fine.

Though, `import`ing `conftest` directly, instead of just setting up a bunch of fixtures is a weird way of doing things.  But I don't think this PR is the place to address that.

**Pull request merged**